code/modules/projectiles/guns/smartgun.dm

```diff
@@ -317,13 +317,16 @@
 		var/mob/living/carbon/human/H = user
 		if(!skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_SMARTGUN) && !skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_ALL))
 			to_chat(H, SPAN_WARNING("You don't seem to know how to use \the [src]..."))
```

Comment:
Should these still be here w/ the balloon alerts?

Reply:
I'll remove everything other than the "feed cover open" one because the balloon alert isn't enough to tell people to alt click to close it again.